README.md

```diff
@@ -109,10 +104,10 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
 you have to use a compatible API version.
 
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->2.0.0-alpha<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->2.1.0-alpha<!--SPLUNK_VERSION--> is compatible
 with:
 
-* OpenTelemetry API version <!--OTEL_VERSION-->1.34.1<!--OTEL_VERSION-->
+* OpenTelemetry API version <!--OTEL_VERSION-->1.35.0<!--OTEL_VERSION-->
 * OpenTelemetry Instrumentation for Java version <!--OTEL_INSTRUMENTATION_VERSION-->2.0.0<!--OTEL_INSTRUMENTATION_VERSION-->
```

Comment:
Weird that this didn't get updated.
```suggestion
* OpenTelemetry Instrumentation for Java version <!--OTEL_INSTRUMENTATION_VERSION-->2.1.0<!--OTEL_INSTRUMENTATION_VERSION-->
```

Reply:
thanks, I messed up the arguments for the pre release script